### ISSUE TITLE
Version Lock Morph Module for Live Radio

### ISSUE THREAD

**Issue by ryanmccombe** [bbc/simorgh#4795]
**Is your feature request related to a problem? Please describe.**
We currently have a small production risk around live radio

We use an “embed endpoint” in Mozart.  This page uses a morph module, however, it does not specify which version of that morph module.

If someone deploys a new version of the morph module, our embed endpoint will automatically start using that version, without us having the opportunity to test it

**Describe the solution you'd like**
Version lock the morph module used within our page config.

The page config is here: https://github.com/bbc/mozart-page-configs/blob/master/configs/ws/av_embeds_media.json

An example of how the version lock is applied is here: https://github.com/bbc/mozart-page-configs/pull/1292

The morph module is here (for reference - no changes necessary): https://github.com/bbc/morph-modules/tree/master/news-av-player

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

**Comment by aeroplanejane** [comment]
@ryanmccombe is this required before we can go live?

**Comment by ryanmccombe** [comment]
> @ryanmccombe is this required before we can go live?

This is a live radio issue - added to wrong milestone

**Comment by ryanmccombe** [comment]
Released to TEST

**Comment by ryanmccombe** [comment]
Released to LIVE

**Comment by ryanmccombe** [comment]
This is already done
